packages/fluentui/e2e/tests/tooltipDismissOnHoverContent.ts

```diff
@@ -0,0 +1,20 @@
+import { selectors } from './tooltipDismissOnHoverContent-example';
+
+const trigger = `.${selectors.triggerClassName}`;
+const content = `.${selectors.contentClassName}`;
+
+describe('Tooltip', () => {
+  beforeEach(() => {
+    cy.gotoTestCase(__filename, trigger);
+  });
+
+  it('Should close on content hover', () => {
+    cy.clickOn(trigger);
+
+    cy.exist(content);
+
+    cy.hover(content);
+
+    cy.wait(15).notExist(content);
```

Comment:
```suggestion
    cy.notExist(content);
```

We should avoid usage of `.wait()`, otherwise our tests will be flacky on CI. Is it possible?

Reply:
We have a default delay for mouse close of 10 ms

Reply:
Then let's set it to zero for this example to make tests more predictable